opensearch/src/test/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/MatchPhraseQueryTest.java

```diff
@@ -14,6 +14,7 @@
 import org.junit.jupiter.api.DisplayNameGenerator;
 import org.junit.jupiter.api.Test;
 import org.opensearch.sql.common.antlr.SyntaxCheckException;
+import org.opensearch.sql.common.grok.Match;
```

Comment:
This can be removed as it is not being used.

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/165/commits/38b7714eacf6004acda53d18c2b65dc3ae135bc5